distribution/docs/src/main/jdocs/content/_plugins/point-of-contact-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,20 @@
+:type: plugin
+:status: published
+:title: Point of Contact Policy Plugin
+:link: _point_of_contact_policy_plugin
+:plugintypes: policy
+:summary: Adds a policy if Point of Contact is updated.
+
+The Point of Contact Policy Plugin adds a policy if the Point-of-Contact field is changed during pre-update.
```

Comment:
✏️  "...pre-update, thus rejecting the update operation."? Readers may not understand what "adding to the policy" really does here.